src/plugins/timeConductor/ConductorAxis.vue

```diff
@@ -55,20 +57,69 @@ export default {
     }
   },
   emits: ['pan-axis', 'end-pan', 'zoom-axis', 'end-zoom'],
+  setup() {
+    const axisHolder = ref(null);
+    const { size: containerSize, startObserving } = useResizeObserver();
+
+    onMounted(() => {
+      startObserving(axisHolder.value);
+    });
+
+    return {
+      axisHolder,
+      containerSize
+    };
+  },
   data() {
     return {
       inPanMode: false,
       dragStartX: undefined,
       dragX: undefined,
-      zoomStyle: {}
+      zoomStyle: {},
+      rect: undefined
```

Comment:
picky, take it or leave, but explicitly defining things as undefined seems weird. can we use null here?

Reply:
It is weird! and I hate doing it. But sometimes switching these things to `null` cascades weird issues down the line where we explicitly check for `undefined`. I think we should be safe to update this one, though.